api/v1beta1/types_common.go

```diff
@@ -0,0 +1,141 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import corev1 "k8s.io/api/core/v1"
+
+// AzureClusterCommon defines the azureCluster properties shared across all azure clusters.
```

Comment:
> shared across all azure clusters

potentioally misleading? maybe 

```suggestion
// AzureClusterCommon defines the AzureCluster properties that may be shared across several AzureClusters.
```